typescript/src/serviceyaml.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export interface ServiceYaml {
+  title: string;
+  apis: string[];
```

Comment:
At some point, maybe not in this PR as it is rather focused, you will want the `http` rules as well.

Reply:
Good to know! Created a quick issue for it here: https://github.com/googleapis/gapic-generator-typescript/issues/929 and also updated the `README` with the new `bazel run` command to include the `$service.yaml` file.

Reply:
Oh and chatted with Alex and Summer about the `iamService` flag - we decided that we'll go ahead with removing the `iamService` flag and bump the generator version (since it is technically a breaking change), and then update the `//third_party/googleapis/WORKSPACE`, `//third_party/googleapis_private/WORKSPACE`, and the `BUILD` files for `kms` and `pubsub` to use the latest generator version which should cover all the protos currently using `IAMPolicy`.